src/templates/pages/application.html

```diff
@@ -20,7 +20,10 @@ <h2 class="text-3xl text-center">Contact and address</h2>
             <div>
                 {% include 'components/dropdown.html' with classes='w-full bg-[#6C37BD] bg-opacity-30' %}
             </div>
-            <div class="mt-5">
+            <div>
```

Comment:
Remove this line. Postal code is not required field, since we have postal code on the City, when you are selecting city it has postal code @tinnamariat